### PR TITLE
[holo-rsm]: Bundled Core-App

### PR DESCRIPTION
**Pull request by zo-el**

### CODE REVIEW

**Comment by alastairong** [comment]
@JettTech FYI looks like Joel has created a core-app bundle

File: profiles/logical/holo/match-server/default.nix [hunk 1]
```diff
@@ -41,19 +41,31 @@ in
...
+           }
+           {
+             id = "holofuel";
+             url = "https://holo-host.github.io/holo-hosting-app-rsm/releases/downloads/holofuel/0_1_0_alpha0/holofuel.0_1_0_alpha0.dna";
+             # Do not change the linting for these core-app properties
+             # properties need to be a valid yaml string, which means you have to be specific with spaces and tabs
+             properties =''
+skip_proof: ${holoNetwork.memProofServer.coreAppProperties.skipProof}
+holo_agent_override: "${holoNetwork.memProofServer.coreAppProperties.holoAgentOverride}"
+not_editable: "${holoNetwork.memProofServer.coreAppProperties.holoAgentOverride}"
```
Comment (robbiecarlton) [inline profiles/logical/holo/match-server/default.nix:64]:
```suggestion
not_editable_profile: true
```

File: profiles/logical/hpos/default.nix [hunk 2]
```diff
@@ -297,19 +297,31 @@ in
...
-         properties =''
+           {
+             id = "holofuel";
+             url = "https://holo-host.github.io/holo-hosting-app-rsm/releases/downloads/holofuel/0_1_0_alpha0/holofuel.0_1_0_alpha0.dna";
+             # Do not change the linting for these core-app properties
+             # properties need to be a valid yaml string, which means you have to be specific with spaces and tabs
+             properties =''
 skip_proof: ${holoNetwork.memProofServer.coreAppProperties.skipProof}
 holo_agent_override: "${holoNetwork.memProofServer.coreAppProperties.holoAgentOverride}"
+not_editable: "${holoNetwork.memProofServer.coreAppProperties.holoAgentOverride}"
```
Comment (robbiecarlton) [inline profiles/logical/hpos/default.nix:320]:
```suggestion
not_editable_profile: true
```

**Review by robbiecarlton — APPROVED** [review]
just the editable change

**Pull request merged**